lib/createlocaltracks.js

```diff
@@ -55,6 +55,30 @@ let createLocalTrackCalls = 0;
  *     console.log(localTrack.name);
  *   });
  * });
+ *
+ * @example
+ * // On mobile browsers, the camera can be reserved by only one {@link LocalVideoTrack}
+ * // at any given time. If you attempt to create a second {@link LocalVideoTrack},
+ * // video frames will no longer be supplied to the first {@link LocalVideoTrack}.
+ * // So, it is recommended that If you want to display your camera preview,
+ * // pre-acquire media using createLocalTracks. You can then pass these LocalTracks
+ * // to connect.
```

Comment:
```suggestion
 * // If you want to display your camera preview, pre-acquire media using createLocalTracks.
 * // You can then pass these LocalTracks to connect().
```

Reply:
sure, will move the blurb in description.